woof-code/rootfs-petbuilds/fixmenusd/fixmenusd.c

```diff
@@ -61,6 +110,9 @@ handle_events(const int fd)
 			    (strcmp(&event->name[len - (sizeof(".desktop") - 1)], ".desktop") != 0))
 				continue;
 
+			if (!(event->mask & IN_DELETE))
+				setup_spot(re, event->name, set);
```

Comment:
Bug here: we don't know if the executable is already there when the .desktop file is created